### PR TITLE
Fix 700

### LINKED ISSUE #700: VLC won't minimize and have title
**What version of WebTorrent Desktop?** (See the 'About WebTorrent' menu)
Version 0.8.1 (0.95.2)
**What operating system and version?**
Ubuntu 16.04
**What did you do?**
Open unsupported format in vlc
**What did you expect to happen?**
vlc to have title displaying played file in top bar rather than localhost and minimize player
**What actually happened?**
played file isn't shown, and it won't minimize (vlc).

### PR DESCRIPTION
**Pull request by feross**
Fix #700

The title is set with the `--meta-title` flag to VLC.

### CODE REVIEW

File: src/main/dialog.js [hunk 1]
```diff
@@ -118,5 +117,5 @@ function setTitle (title) {
 }
 
 function resetTitle () {
-  setTitle(config.APP_WINDOW_TITLE)
+  window.main.dispatch('resetTitle')
```
Comment (mathiasvr) [inline src/main/dialog.js]: Correction: `window` should be `windows`

**Comment by mathiasvr** [comment]
LGTM aside from comment, nice use of `dispatch` 👍

File: src/main/dialog.js (hunk 1 above)
Comment (feross) [inline src/main/dialog.js]: Good catch!

**Comment by feross** [comment]
@mathiasvr thanks for the review!

**Pull request merged**